### PR TITLE
data: Charchitkumawat04

### PR DESCRIPTION
**Pull request by Charchitkumawat04**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

    Changes Made

## Note to reviewers

Just Started with my Open Source. Pls accept me as Your Member.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2871"></a>

### CODE REVIEW

File: data/CharchitKumawat.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "name": "Charchit kumawat",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Self Learner**. Web 3 enthusiast & aspiring businessman",
+  "avatar": "https://github.com/charchitkumawat04.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Charchit_Web3",
+      "icon": "FaTwitter",
```
Comment (Aadarsh805) [inline data/CharchitKumawat.json:11]: Hey this comma at the end of line 11 needs to be removed

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Review by Charchitkumawat04 — COMMENTED** [review]
comma remove

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

Please fix your JSON profile naming. It should be `{your GitHub username}.json -> `charchitkumawat04.json` (capitalization issue)

If you need any help feel free to ask.

**Review by Charchitkumawat04 — COMMENTED** [review]
file name changed to github user name

**Review by Charchitkumawat04 — COMMENTED** [review]
file name updated

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**